source/quickstart.rst

```diff
@@ -9,22 +9,32 @@ Welcome to VAlgoLang's documentation!
 VAlgoLang is a domain specific language that helps educators create animations for teaching students about data structures
 and algorithms.
 
-Here is an example:
+Here's a short example:
 
 .. code:: javascript
 
-    let stack: Stack = new Stack;
+    let stack = Stack<number>();
     stack.push(5);
-    stack.pop();
+    let x: number = stack.pop();
 
 This would produce the video:
 
 .. raw:: html
 
         <video src="_static/intro.mp4" frameborder="0" allowfullscreen style=" width: 80%; height: 80%;" controls></video>
 
-Quick start
------------
 
-#. Step 1
-#. Step 2
+
+Quick Start Guide
+-------------------
+
+The quickest way to get started with VAlgoLang is to head over to the **web interface** at `https://valgolang.netlify.app/ <https://valgolang.netlify.app/>`_. Take a look at the examples pane on the left hand side to get an idea of what you can do with the language and just get stuck in!
+
+Alternatively, if you'd prefer to work locally, follow `these instructions <https://github.com/ManimDSL/ManimDSLCompiler#installation>`_ to install the required dependencies on your machine. The file extension for VAlgoLang files is ``.val`` .
+
+The next steps are to:
+
+#. Take a look through our :doc:`language tour <tutorial>` to familiarise yourself with the syntax of VAlgoLang. It should be pretty easy to pick up if you have even a bit of programming experience!
```

Comment:
Since we capitalize the "Customising Your Animation" maybe we can capitalize "Language Tour" here too?